1-bootstrap/outputs.tf

```diff
@@ -38,3 +38,8 @@ output "source_repo_urls" {
   description = "Source repository URLs"
   value       = { for repo_id, repo in google_sourcerepo_repository.gcp_repo : repo_id => "https://source.developers.google.com/p/${var.project_id}/r/${repo.name}" }
 }
+
+output "cb_service_accounts_emails" {
+  description = "Service Accounts for the Multitenant Administration Cloud Build Triggers"
```

Comment:
Do we want to use `Multi-tenant` or `Multitenant`?  Currently there is a mix.

Reply:
Let's stick with `multitenant`